src/main/java/mate/academy/service/UserService.java

```diff
@@ -8,13 +8,25 @@ public class UserService {
             new User("alice@i.ua", "1234")
     };
 
+    public static User[] getUsers() {
+        return users;
+    }
+
     /**
      * Find user by email. All users are stored in <code>private static final User[] users</code>
      * @param email - the input parameter
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
     public User findByEmail(String email) {
+
```

Comment:
```suggestion
```

Reply:
not fixed. Redundant empty line should be deleted